### PR TITLE
WIP: Medical facility

### PR DESCRIPTION
**Pull request by unjown**
**Content (Artwork / Missions / Jobs)**

Medical facility missions (will be adding more)
Medical facility outfit 

## Summary
I have added a new outfit medical facility mainly used for missions

## PR Checklist
 - [X] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [ ] I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}
 - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}
  
  
-----------------------
**Feature:** This PR implements the feature request detailed and discussed in issue #{{6066}}

## Feature Details
I have added a new outfit the medical facility mainly for missions for now

## Testing Done
N/A yet

PS: help needed in making the outfit image thx

### CODE REVIEW

File: data/human/near earth jobs.txt [hunk 1]
```diff
@@ -157,3 +157,22 @@ mission "From Earth Day celebration [1]"
...
+	source
+		government "Republic"
+		not system "Sol"
+	destination
+		"Earth"
+	on offer 
+		require "Medical Facility"
+	on complete
+		payment 50000
+		dialog `Your passenger's personal nurse thanks you for transporting him`
```
Comment (Pointedstick) [inline data/human/near earth jobs.txt:178]:
```suggestion
		dialog `Your passenger's nurse thanks you for transporting them and hands you your payment of <payment>.`
```

File: data/human/near earth jobs.txt [hunk 2]
```diff
@@ -157,3 +157,22 @@ mission "From Earth Day celebration [1]"
 		payment
 		payment 1000
 		dialog `Your passengers thank you for returning them to <planet>, and you collect your payment of <payment>.`
+mission "Transport patient [1]"
+	name `Transport patient"
+	job
+	repeat
+	description `Transfer a patient with his personal nurse to <destination> for better medical care. Payment is <payment>`
```
Comment (Pointedstick) [inline data/human/near earth jobs.txt:164]:
```suggestion
	description `Transfer a patient with their personal nurse to <destination> for better medical care. Payment is <payment>`
```

File: data/human/outfits.txt [hunk 3]
```diff
@@ -330,6 +330,16 @@ outfit "Luxury Accommodations"
...
 
+outfit "Medical Facility"
+	category "Systems"
+	cost 100000
+	thumbnail "outfit/Medical facility"
+	"mass" 7
+	"outfit space" -7
+	"required crew" 2
+	"unplunderable" 1
+	description ""
```
Comment (Pointedstick) [inline data/human/outfits.txt:341]: Needs a description, obvs :)

**Review by Pointedstick — CHANGES_REQUESTED** [review]
Consider adding a maximum distance and other relevant filters.

**Comment by MasterOfGrey** [comment]
Have you considered the possibility of a Major Medical Facility that could give you missions to respond to humanitarian crises'?

**Comment by Pointedstick** [comment]
No response to requested changes and PR is in an early alpha state; closing for now to un-clutter the open PR list. @unjown, we can re-open this if you would like to resume work on it.

**Pull request closed without merging**